src/ofxstatement/plugins/alfabank.py

```diff
@@ -0,0 +1,128 @@
+#    AlfaBank (https://www.alfabank.ru) plugin for ofxstatement
+#
+#    Copyright 2017 Dmitry Pavlov <zeldigas@gmail.com>
+#
```

Comment:
I thought that only author is listed and I don't see contributions  ;).  if you think that plugin structure is a good reason to add you here (of course I took it from tinkoff one to follow single coding style) I'm fine to add you